payments_service/braintree/forms.py

```diff
@@ -17,6 +17,10 @@ class SubscriptionForm(forms.Form):
     pay_method_uri = forms.CharField(max_length=255, required=False)
     # Braintree subscription plan ID.
     plan_id = forms.CharField(max_length=255)
+    # A custom amount to pay for the subscription. This only applies to
+    # things like recurring donations. Solitude validates this value to make
+    # sure you can't adjust fixed price subscriptions.
+    amount = forms.DecimalField(required=False)
```

Comment:
hmm, maybe. I hesitate to stray from the solitude API fields unless there's a really good reason.